runtime/src/bank/partitioned_epoch_rewards/calculation.rs

```diff
@@ -298,21 +298,6 @@ impl Bank {
             cached_vote_accounts,
         } = reward_calculate_params;
 
-        let solana_vote_program: Pubkey = solana_vote_program::id();
-
-        let get_vote_account = |vote_pubkey: &Pubkey| -> Option<VoteAccount> {
-            if let Some(vote_account) = cached_vote_accounts.get(vote_pubkey) {
-                return Some(vote_account.clone());
-            }
-            // If accounts-db contains a valid vote account, then it should
```

Comment:
it looks like this comment exists in 3 places.

Reply:
yeah. We copy/paste `get_vote_account several in two other places. One of them is in test. I will update the.